scitbx/graph/tardy_tree.py

```diff
@@ -243,7 +244,9 @@ def cmp_elems(a, b):
       if (fa < fb): return 1
       if (a[1] > b[1]): return -1
       if (a[1] < b[1]): return 1
-      return cmp(a[0], b[0])
+      if (a[0] > b[0]): return 1
+      if (a[0] < b[0]): return -1
+      return 0
```

Comment:
Duplicate `cmp`
Change the import to `from libtbx.math_utils import cmp` so nothing else needs to be changed.